flows/expressions_test.go

```diff
@@ -0,0 +1,15 @@
+package flows_test
+
+import (
+	"testing"
+
+	"github.com/nyaruka/goflow/flows"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestContactQueryEscaping(t *testing.T) {
+	assert.Equal(t, `""`, flows.ContactQueryEscaping(``))
+	assert.Equal(t, `"bobby tables"`, flows.ContactQueryEscaping(`bobby tables`))
+	assert.Equal(t, `"\"\" OR (id = 1)"`, flows.ContactQueryEscaping(`"" OR (id = 1)`))
```

Comment:
Can the input contain an escape?

IE, what happens with a query like `name = "\"foo"` Wouldn't that turn into `name = "\\"foo"`?

Reply:
that would become `"\\\"foo"`.. escaped `\` and escaped `"`, interpreted by the query parser as the value `\"foo`